### PR TITLE
yabai: add page

### PR DESCRIPTION
**Pull request by BosEriko**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/yabai.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> A tiling window manager for macOS based on binary space partitioning.
+> More information: <https://github.com/koekeishiya/yabai>.
+
+- Set the layout to bsp:
+
+`yabai -m config layout bsp`
```
Comment (zdroid) [inline pages/osx/yabai.md]:
```suggestion
`yabai -m config layout {{bsp}}`
```

File: pages/osx/yabai.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set the window gap:
+
+`yabai -m config window_gap 10`
```
Comment (zdroid) [inline pages/osx/yabai.md]:
```suggestion
`yabai -m config window_gap {{10}}`
```

File: pages/osx/yabai.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> A tiling window manager for macOS based on binary space partitioning.
+> More information: <https://github.com/koekeishiya/yabai>.
+
+- Set the layout to bsp:
+
+`yabai -m config layout bsp`
+
+- Set the window gap:
```
Comment (zdroid) [inline pages/osx/yabai.md]:
```suggestion
- Set the window gap to 10:
```

Maybe even `10 pixels` (?) if that's the correct unit of measure here.

Comment (BosEriko) [reply pages/osx/yabai.md]: I've updated it to `to 10pt` instead. Based on the looks it's definitely pixels but on Yabai's documentation they didn't mention it as pixels, they did mention it as 10pt, though.

I've also updated the other lines based on your comments. Thanks!

File: pages/osx/yabai.md [hunk 4]
```diff
@@ -5,11 +5,11 @@
 
 - Set the layout to bsp:
 
-`yabai -m config layout bsp`
+`yabai -m config layout {{bsp}}`
 
-- Set the window gap:
+- Set the window gap to 10pt:
```
Comment (zdroid) [inline pages/osx/yabai.md]: Is it pt or px? Nevertheless, it should be in this form:
```suggestion
- Set the window gap to 10 pt:
```

Comment (mebeim) [reply pages/osx/yabai.md]: The unit is usually not spaced (see any program that takes values like `10px` `10G`, `10K`). I would leave it as is.

Comment (zdroid) [reply pages/osx/yabai.md]: @mebeim Right, I deleted that part.

Comment (BosEriko) [reply pages/osx/yabai.md]: It seems to be like it's pixels but on their documentation, they call it pt. That's why I went with pt. Shall I proceed with using px for the sake of unity between instructions here in tldr?

Comment (mebeim) [reply pages/osx/yabai.md]: I think just saying "pixels" is the best choice here. After all, you said they call it "pixels" in their documentation too, right?

Comment (mebeim) [reply pages/osx/yabai.md]: @BosEriko I can't find where exactly they use `pt`. Are you talking about [this](https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc) documentation?

Comment (BosEriko) [reply pages/osx/yabai.md]: I found it here.
https://github.com/koekeishiya/yabai/wiki/Configuration#padding-and-gaps

Comment (mebeim) [reply pages/osx/yabai.md]: That's strange. I suppose they say `pt` to emphasize the fact that display scaling could result in different real pixel values. I don't see any mention of "pixels" in the docs though, I think we can leave it as is.

Comment (sbrl) [reply pages/osx/yabai.md]: No idea what this tool is, but a `pt` and a `px` are not actually the same thing. `px` is a pixel, and `pt` is a point - which may be a different size to a pixel IIRC.

Comment (zdroid) [reply pages/osx/yabai.md]: You are correct on using `pt` here. Thanks for contributing!

File: pages/osx/yabai.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`yabai -m config layout {{bsp}}`
+
+- Set the window gap to 10pt:
+
+`yabai -m config window_gap {{10}}`
+
+- Enable opacity:
+
+`yabai -m config window_opacity on`
```
Comment (agnivade) [inline pages/osx/yabai.md]: on and off should be tokens (`{{}}`)

Comment (zdroid) [reply pages/osx/yabai.md]: It could be, but only if `enable` was changed to `enable/disable`.

Comment (agnivade) [reply pages/osx/yabai.md]: Enable/disable would need it to be {{on|off}}. But if it's just enable, then I think it should just be {{on}}

**Pull request merged**